lib/main.dart

```diff
@@ -1,115 +1,185 @@
 import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'TDL_provider.dart';
 
-void main() {
-  runApp(const MyApp());
+void main() => runApp(const TaskApp());
+
+class Task {
+  Task({required this.name, required this.checked});
+  final String name;
+  bool checked;
+}
+
+class TaskList extends StatefulWidget {
+  const TaskList({Key? key}) : super(key: key);
+
+  get checked => null;
+
+  @override
+  _TaskListState createState() => _TaskListState();
 }
 
-class MyApp extends StatelessWidget {
-  const MyApp({Key? key}) : super(key: key);
+class _TaskListState extends State<TaskList> {
+  final TextEditingController _textFieldController = TextEditingController();
+  final List<Task> _Tasks = <Task>[];
 
-  // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-      title: 'Flutter Demo',
-      theme: ThemeData(
-        // This is the theme of your application.
-        //
-        // Try running your application with "flutter run". You'll see the
-        // application has a blue toolbar. Then, without quitting the app, try
-        // changing the primarySwatch below to Colors.green and then invoke
-        // "hot reload" (press "r" in the console where you ran "flutter run",
-        // or simply save your changes to "hot reload" in a Flutter IDE).
-        // Notice that the counter didn't reset back to zero; the application
-        // is not restarted.
-        primarySwatch: Colors.blue,
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Task manager: TIG333'),
+        backgroundColor: const Color.fromARGB(255, 0, 0, 0),
```

Comment:
Ett lite mer läsbart alterantiv:
```suggestion
        backgroundColor: Colors.black,
```